extensions/amp-viewer-integration/0.1/amp-viewer-integration.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isExperimentOn} from '../../../src/experiments';
+import {user} from '../../../src/log';
+
+const TAG = 'amp-viewer-integration';
+
+
+export class AmpViewerIntegration extends AMP.BaseElement {
```

Comment:
No need for an element, since we won't have any real extensions here. You may choose to install a global service or an ampdoc service. See `AMP.registerServiceForDoc` - that depends on how much state you will have.